src/iconsManifest/supportedExtensions.ts

```diff
@@ -82,7 +82,13 @@ export const extensions: IFileCollection = {
     { icon: 'ai2', extensions: ['ai'], format: FileFormat.svg, disabled: true },
     {
       icon: 'al',
-      extensions: [],
+      extensions: ['al'],
```

Comment:
Please, remove this extension as we want this to be manage by the language id. So every extension creator should be responsible for adding support to the extensions.